---
layout: page
title: People
permalink: /people/
---

# Current Members

<div class="center">

<div class="item">
    <img class="rcornersimage" src="/assets/images/sebastian.jpg">
    <h5><div class="tooltip"><p class="ttext">Sebastian Höhna</p>
        <span class="tooltiptext">
            I’m a computational biologist with focus on Bayesian phylogenetic inference. Natural selection by means of adaptation and genetic inheritance are key principles in evolutionary biology. Evolutionary relationships are commonly depicted by phylogenetic trees. Using phylogenetic methods we can learn about the evolutionary history of species and the processes that have contributed to present diversity. I’m developing statistical and computational methods to infer phylogenies from molecular sequence data. These methods additionally identify periods of adaptive genetic evolution at lineage or genes. Furthermore, I develop mathematical models to study macroevolutionary patterns, such as, diversification rate variation over time and among lineages and episodes of global mass extinctions.
        </span>
    </div></h5>
        <p class="title"><em>Group Leader</em></p>
        <a href="https://github.com/hoehnalab/hoehnalab.github.io/raw/master/CVs/CV_Sebastian.pdf" target="_blank"><img src="/assets/icons/cv.png" width="25px"></a>
        <a href="https://github.com/hoehna" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="https://twitter.com/sebastianhoehna?lang=en" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>
        <a href="mailto:hoehna@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
        I’m a computational biologist with focus on Bayesian phylogenetic inference. Natural selection by means of adaptation and genetic inheritance are key principles in evolutionary biology. Evolutionary relationships are commonly depicted by phylogenetic trees. Using phylogenetic methods we can learn about the evolutionary history of species and the processes that have contributed to present diversity. I’m developing statistical and computational methods to infer phylogenies from molecular sequence data. These methods additionally identify periods of adaptive genetic evolution at lineage or genes. Furthermore, I develop mathematical models to study macroevolutionary patterns, such as, diversification rate variation over time and among lineages and episodes of global mass extinctions.
    </div>
</div>
    
    
<div class="item">
    <img class="rcornersimage" src="/assets/images/John_Clarke.jpg">
    <h5><div class="tooltip"><p class="ttext">John Clarke</p>
        <span class="tooltiptext">
I'm an evolutionary biologist and palaeontologist interested in testing drivers of phenotypic and species diversity using phylogenetic comparative methods. Having first applied these methods to Mesozoic fishes, I now apply them to large datasets of extant fishes, plants and insects.
        </span>
    </div></h5>
        <p class="title"><em>Postdoc (Humboldt fellow)</em></p>
        <a href="mailto:j.clarke.paleo@gmail.com"><img src="/assets/icons/mail.svg" width="25px"></a>    
        <a href="https://twitter.com/jclarkepaleo?lang=en" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I'm an evolutionary biologist and palaeontologist interested in testing drivers of phenotypic and species diversity using phylogenetic comparative methods. Having first applied these methods to Mesozoic fishes, I now apply them to large datasets of extant fishes, plants and insects.
    </div>
</div>
    
    
<div class="item">
    <img class="rcornersimage" src="/assets/images/Alessio_Capobianco.jpg">
    <h5><div class="tooltip"><p class="ttext">Alessio Capobianco</p>
        <span class="tooltiptext">
I am a paleontologist and evolutionary biologist investigating how biological diversity evolves in concert with changes in the Earth system, mainly focusing on fishes. My research explores how fossils reveal fundamental and entirely unexpected aspects of the evolutionary history of living organisms, and examines diversity and biogeographic patterns through the lens of paleontological data. I am also interested in how sampling and coding of morphological data impacts phylogenetic inference and the estimation of evolutionary timescales.
        </span>
    </div></h5>
        <p class="title"><em>Postdoc</em></p>
        <a href="mailto:acapo@umich.edu"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I am a paleontologist and evolutionary biologist investigating how biological diversity evolves in concert with changes in the Earth system, mainly focusing on fishes. My research explores how fossils reveal fundamental and entirely unexpected aspects of the evolutionary history of living organisms, and examines diversity and biogeographic patterns through the lens of paleontological data. I am also interested in how sampling and coding of morphological data impacts phylogenetic inference and the estimation of evolutionary timescales.
    </div>
</div>
    
<div class="item">
    <img class="rcornersimage" src="/assets/images/Nicola.jpg">
    <h5><div class="tooltip"><p class="ttext">Nicola Heckeberg</p>
        <span class="tooltiptext">
As an evolutionary palaeobiologist, I am interested in morphological character changes of vertebrates, especially mammals, through time.
Functional trait evolution in ruminants and other artiodactyls are the focus of my research, integrating fossil and extant taxa and molecular and phenotypic data in comprehensive time calibrated analyses.
        </span>
    </div></h5>
        <p class="title"><em>Postdoc</em></p>
        <a href="mailto:n.heckeberg@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>
        <a href="https://twitter.com/heckeberg" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>


    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
As an evolutionary palaeobiologist, I am interested in morphological character changes of vertebrates, especially mammals, through time.
Functional trait evolution in ruminants and other artiodactyls are the focus of my research, integrating fossil and extant taxa and molecular and phenotypic data in comprehensive time calibrated analyses.
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/Gustavo.JPEG">
    <h5><div class="tooltip"><p class="ttext">Gustavo Darlim</p>
        <span class="tooltiptext">
I’m a vertebrate paleontologist working on macroevolution of crocodylomorphs (group that also includes living crocodilians). My research is dedicated to a critical reassessment of paleontological and neontological data to be subsequently incorporated into analyses that combine morphological, molecular and stratigraphic (fossil age) data simultaneously (total-evidence tip dating). The focus of my studies is to provide more rigorous time-scaled phylogenies in order to better understanding macroevolutionary processes underlying crocodylomorph evolutionary history.
        </span>
    </div></h5>
        <p class="title"><em>Postdoc</em></p>
        <a href="mailto:g.darlim@lrz.uni-muenchen.de"><img src="/assets/icons/mail.svg" width="25px"></a>
        <a href="https://twitter.com/GDarlim" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I’m a vertebrate paleontologist working on macroevolution of crocodylomorphs (group that also includes living crocodilians). My research is dedicated to a critical reassessment of paleontological and neontological data to be subsequently incorporated into analyses that combine morphological, molecular and stratigraphic (fossil age) data simultaneously (total-evidence tip dating). The focus of my studies is to provide more rigorous time-scaled phylogenies in order to better understanding macroevolutionary processes underlying crocodylomorph evolutionary history.
    </div>
</div>
    
<div class="item">
    <img class="rcornersimage" src="/assets/images/luiza.jpg">
    <h5><div class="tooltip"><p class="ttext">Luiza Fabreti</p>
        <span class="tooltiptext">
            I'm interested in computational and evolutionary biology. Specially, improving the way we model evolution. My research involves Bayesian phylogenetic inference and underlying topics such as, model selection, testing the adequacy of the most commonly used substitution models and assessing convergence of the Markov Chain Monte Carlo.
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="https://github.com/lfabreti" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="mailto:l.fabreti@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
        I'm interested in computational and evolutionary biology. Specially, improving the way we model evolution. My research involves Bayesian phylogenetic inference and underlying topics such as, model selection, testing the adequacy of the most commonly used substitution models and assessing convergence of the Markov Chain Monte Carlo.
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/ronja.jpg">
    <h5><div class="tooltip"><p class="ttext">Ronja Billenstein</p>
        <span class="tooltiptext">
            I’m a biologist working on statistical methods for inferring the evolutionary history of closely related populations. These methods are based on coalescent theory and a mutation process. I am using simulated as well as empirical data to test and to validate these methods, hoping to provide an accurate, powerful and computationally feasible approach.
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="mailto:r.billenstein@lrz.uni-muenchen.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
        I’m a biologist working on statistical methods for inferring the evolutionary history of closely related populations. These methods are based on coalescent theory and a mutation process. I am using simulated as well as empirical data to test and to validate these methods, hoping to provide an accurate, powerful and computationally feasible approach.
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/killian.jpg">
    <h5><div class="tooltip"><p class="ttext">Killian Smith</p>
        <span class="tooltiptext">
I am a computer scientist working with Bayesian phylogenetics and substitution models, and am
interested in applying concepts and theory from CS in this domain. In my research I will be
exploring covarion models, mixture models, and Markov modulated continuous Markov chains (MMCMC),
and their effects on phylogenetic tree construction. Some of my other research interests include
type theory, functional programming, de novo sequencing, multiple sequence alignment, genome
assembly, and machine learning.
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="mailto:k.smith@lrz.uni-muenchen.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I am a computer scientist working with Bayesian phylogenetics and substitution models, and am
interested in applying concepts and theory from CS in this domain. In my research I will be
exploring covarion models, mixture models, and Markov modulated continuous Markov chains (MMCMC),
and their effects on phylogenetic tree construction. Some of my other research interests include
type theory, functional programming, de novo sequencing, multiple sequence alignment, genome
assembly, and machine learning.
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/bjorn.jpg">
    <h5><div class="tooltip"><p class="ttext">Bjørn T. Kopperud</p>
        <span class="tooltiptext">
I am an evolutionary biologist who is interested in estimating speciation- and extinction rates from phylogenetic trees. In my research, I have been investigating the challenge associated with non-identifiability in the time-variable reconstructed birth-death model, as presented by Louca & Pennell (2020) in Nature. Specifically, we have found that rapidly changing speciation and extinction rates can be robustly inferred in spite of non-identifiability. Additionally, I am working on developing a fast algorithm for a lineage-specific birth-death model with finite categories, which is used to infer shifts in rates along the phylogeny. Previously, this model has relied on Monte Carlo methods for estimating the rate and number of shifts along each branch. With our new approach, we do not need to use the Monte Carlo approach. Consequently, our method is much faster, there is no stochasticity in the estimation procedure, and we can readily fit the lineage-specific birth-death model to large trees with tens of thousands of taxa. My other interests include data mining using natural language processing, brain size evolution and phylogenetic comparative models of adaptation. My favorite organisms are bryozoans and ruminants!
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="https://github.com/kopperud" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="mailto:b.kopperud@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I am an evolutionary biologist who is interested in estimating speciation- and extinction rates from phylogenetic trees. In my research, I have been investigating the challenge associated with non-identifiability in the time-variable reconstructed birth-death model, as presented by Louca & Pennell (2020) in Nature. Specifically, we have found that rapidly changing speciation and extinction rates can be robustly inferred in spite of non-identifiability. Additionally, I am working on developing a fast algorithm for a lineage-specific birth-death model with finite categories, which is used to infer shifts in rates along the phylogeny. Previously, this model has relied on Monte Carlo methods for estimating the rate and number of shifts along each branch. With our new approach, we do not need to use the Monte Carlo approach. Consequently, our method is much faster, there is no stochasticity in the estimation procedure, and we can readily fit the lineage-specific birth-death model to large trees with tens of thousands of taxa. My other interests include data mining using natural language processing, brain size evolution and phylogenetic comparative models of adaptation. My favorite organisms are bryozoans and ruminants!
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/wenjie.jpg">
    <h5><div class="tooltip"><p class="ttext">Wenjie Zhu 祝雯洁</p>
        <span class="tooltiptext">
I am a master's student in Evolutionary Biology. My research passion lies in computational and statistical phylogenetics, especially species tree inference from molecular data. I am currently working on evaluating the performance and properties of coalescent-based species tree inference methods using simulations. To offset my screen time, I enjoy exploring nature through hiking and botanizing.
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="mailto:Wenjie.Zhu@campus.lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
I am a master's student in Evolutionary Biology. My research passion lies in computational and statistical phylogenetics, especially species tree inference from molecular data. I am currently working on evaluating the performance and properties of coalescent-based species tree inference methods using simulations. To offset my screen time, I enjoy exploring nature through hiking and botanizing.
    </div>
</div>


<div class="item">
    <img class="rcornersimage" src="/assets/images/Basanta.jpg">
    <h5><div class="tooltip"><p class="ttext">Basanta Khakurel</p>
        <span class="tooltiptext">
            I am a computational biologist working mostly in morphological character evolution models. My research involves estimating phylogenetic trees in a more robust manner using more adequate models of evolution. In the past, I have worked on species delimitation studies on Mexican livebearer fishes and Eastern Pine snakes. My work also involved assessing the impact of morphological state space on phylogenetic inferences and trying to allow for different stationary frequencies in morphological character evolution. I will be exploring more on the use of various models for morphological character evolution.
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="https://github.com/basanta33" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="https://basanta33.github.io/" target="_blank"><img src="/assets/icons/globe.svg" width="25px"></a>
        <a href="https://twitter.com/basantakhakurel" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>
        <a href="mailto:b.khakurel@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
            I am a computational biologist working mostly in morphological character evolution models. My research involves estimating phylogenetic trees in a more robust manner using more adequate models of evolution. In the past, I have worked on species delimitation studies on Mexican livebearer fishes and Eastern Pine snakes. My work also involved assessing the impact of morphological state space on phylogenetic inferences and trying to allow for different stationary frequencies in morphological character evolution. I will be exploring more on the use of various models for morphological character evolution.
    </div>
</div>

<div class="item">
    <img class="rcornersimage" src="/assets/images/priscilla.jpg">
    <h5><div class="tooltip"><p class="ttext">Priscilla Lau</p>
        <span class="tooltiptext">
            I am a computational evolutionary biologist interested in macroevolution and trait evolution. I am currently working with Sebastian, Bjørn and John in developing a state-dependent Ornstein-Uhlenbeck model for continuous character evolution in RevBayes. During my undergrad, I studied the phylogenetics of Hawaiian Doryopteris (Pteridaceae) using target enrichment probe set GoFlag. Besides research, I also enjoy curating herbarium specimens, observing linguistics behaviors (sound production and perception), watching baseball games, and playing classic, abstract games (currently minesweeper). 
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="https://github.com/prilau" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="mailto:pri.lau@campus.lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
            I am a computational evolutionary biologist interested in macroevolution and trait evolution. I am currently working with Sebastian, Bjørn and John in developing a state-dependent Ornstein-Uhlenbeck model for continuous character evolution in RevBayes. During my undergrad, I studied the phylogenetics of Hawaiian Doryopteris (Pteridaceae) using target enrichment probe set GoFlag. Besides research, I also enjoy curating herbarium specimens, observing linguistics behaviors (sound production and perception), watching baseball games, and playing classic, abstract games (currently minesweeper). 
    </div>
</div>


<div class="item">
    <img class="rcornersimage" src="/assets/images/walker_profile.jpeg">
    <h5><div class="tooltip"><p class="ttext">Walker Sexton</p>
        <span class="tooltiptext">
            I am a Master's student in the Paleobiology and Geobiology program. My undergraduate research at Washington University in St. Louis focused on developing an empirical hominin phylogeny and investigating the problems that entailed. I am currently working to understand lineage diversification models and developing models to infer phylogeny from morphological data. Outside of the lab, I enjoy traveling, juggling, and exploring the many cultural opportunities Munich has to offer!
            
        </span>
    </div></h5>
        <p class="title"><em>Graduate Student</em></p>
        <a href="https://twitter.com/walkerleesexton" target="_blank"><img src="/assets/icons/twitter.png" width="25px"></a>
        <a href="mailto:Walker.Sexton@lmu.de"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
            I am a Master's student in the Paleobiology and Geobiology program. My undergraduate research at Washington University in St. Louis focused on developing an empirical hominin phylogeny and investigating the problems that entailed. I am currently working to understand lineage diversification models and developing models to infer phylogeny from morphological data. Outside of the lab, I enjoy traveling, juggling, and exploring the many cultural opportunities Munich has to offer!
    </div>
</div>
    
</div>


# Past Members


<div class="center">

<div class="item">
    <img class="rcornersimage" src="/assets/images/allison.jpg">
    <h5><div class="tooltip"><p class="ttext">Allison Hsiang</p>
        <span class="tooltiptext">
            My research uses statistical modeling, Bayesian inference, Big Data, and machine learning to understand morphological evolution and macroevolutionary patterns and processes. I am particularly interested in bringing a ‘next-generation’ approach (e.g., high-throughput automated data generation) to studying fossils and morphology in a phylogenetic context. I am also interested in the interplay between morphology and genetics, including methodological considerations during phylogenetic inference.
        </span>
    </div></h5>
        <p class="title"><em>Independent Researcher (VR Grant) at Stockholm University</em></p>
        <a href="https://github.com/ahsiang" target="_blank"><img src="/assets/icons/mark-github.svg" width="25px"></a>
        <a href="https://ahsiang.github.io/" target="_blank"><img src="/assets/icons/globe.svg" width="25px"></a>
        <a href="mailto:allison.hsiang@gmail.com"><img src="/assets/icons/mail.svg" width="25px"></a>

    <!-- Show tooltip text when on mobile -->
    <div class="blurb">
        My research uses statistical modeling, Bayesian inference, Big Data, and machine learning to understand morphological evolution and macroevolutionary patterns and processes. I am particularly interested in bringing a ‘next-generation’ approach (e.g., high-throughput automated data generation) to studying fossils and morphology in a phylogenetic context. I am also interested in the interplay between morphology and genetics, including methodological considerations during phylogenetic inference.
    </div>

</div>


</div>

