---
layout: page
title: Publications
permalink: /publications/
---
<h3>2026 and in press</h3>
- <strong>Kopperud BT</strong>, <strong>Capobianco A</strong>, <strong>Clarke JT</strong>, Palazzesi L and <strong>Höhna S</strong>. The nature and prevalence of diversification rate shifts across the Tree of Life. *Evolution Letters*. [link](https://doi.org/10.1093/evlett/qrag005)
- <strong>Höhna S</strong>, Freyman W, Nolen Z, Huelsenbeck JP, May MR, Rannala B and Moore BR. Inferring branch-specific rates of lineage diversification under the birth-death-shift process. *Systematic Biology*. [link](https://doi.org/10.1093/sysbio/syag003)
- Wynd BM, <strong>Khakurel B</strong>, Kammerer CF, Wagner PJ, Wright AM. Incorporating continuous characters in joint estimation of dicynodont phylogeny. *Systematic Biology*. [link](https://doi.org/10.1093/sysbio/syaf078)
- <strong>Zhu W</strong>, Chen L, van der Burgt XM, Haba PM, Garaeva D, <strong>Lau P</strong>, Morales-Briones DF, Veranso-Libalah MC. Rediscovery and phylogenomic placement of *Feliciadamia stenocarpa* (Feliciadamieae: Melastomataceae): assessing gene tree discordance. *Botanical Journal of the Linnean Society*. [link](https://doi.org/10.1093/botlinnean/boaf047)
- <strong>Heckeberg NS</strong>, <strong>Capobianco A</strong>, <strong>Khakurel B</strong>, <strong>Darlim G</strong> and <strong>Höhna S</strong>. Practical Guide and Review of Fossil Tip-Dating in Phylogenetics. *Systematic Biology*, 75(1), 156-192. [link](https://doi.org/10.1093/sysbio/syaf050)
- Pienaar J, Bartoszek K, Brahmantio B, Fierst JL, Fuentes-González JA, Hansen TF, Kiang WHC, <strong>Kopperud BT</strong>, Voje KL. Phylogenetic Comparative Methods for Studying Adaptation: The Adaptation-Inertia Framework. *Journal of Evolutionary Biology*, 39 (1), 1-17. [link](https://doi.org/10.1093/jeb/voaf113) 


<h3>2025</h3>
- <strong>Kopperud BT</strong> and <strong>Höhna S</strong>. Phylogenetic Estimation of Branch-specific Shifts in the Tempo of Origination. *Systematic Biology*, 74(6), 985-1006. [link](https://doi.org/10.1093/sysbio/syaf041)
- <strong>Capobianco A</strong> and <strong>Höhna S</strong>. On the Mkv Model with Among-Character Rate Variation. *Systematic Biology*, 74(6), 952-966. [link](https://doi.org/10.1093/sysbio/syaf038)
- <strong>Capobianco A</strong>. How many characters are needed to reconstruct a phylogeny? *Biology Letters*, 21 (10): 20250288. [link](https://doi.org/10.1098/rsbl.2025.0288)
- <strong>Zhu W</strong> and <strong>Höhna S</strong>. Two-step Species Tree Inference under the Multispecies Coalescent Using Full-Likelihood. *Evolutionary Journal of the Linnean Society*, 4(1), kzaf018. [link](https://doi.org/10.1093/evolinnean/kzaf018)
- Paterson RS, Mackie M, <strong>Capobianco A</strong>, <strong>Heckeberg NS</strong>, Fraser D, Demarchi B, Munir F, Patramanis I, Ramos-Madrigal J, Liu S, Ramsøe AD, Dickinson MR, Baldreki C, Gilbert M, Sardella R, Bellucci L, Scorrano G, Leonardi M, Manica A, Racimo F, Willerslev E, Penkman KEH, Olsen JV, MacPhee RDE, Rybczynski N, <strong>Höhna S</strong> and Cappellini E. Phylogenetically-informative proteins from an Early Miocene rhinocerotid. *Nature*, 643 (8072), 719-724. [link](https://doi.org/10.1038/s41586-025-09231-4)
- Blanco F, Lazagabaster IA, Sanisidro Ó, Bibi F, <strong>Heckeberg NS</strong>, Ríos M, Mennecart B, Alberdi MT, Prado SL, Saarinen J, Silvestro D, Müller J, Calatayud J and Cantalapiedra JL. Two major ecological shifts shaped 60 million years of ungulate faunal evolution. *Nature Communications*, 16:4648. [link](https://doi.org/10.1038/s41467-025-59974-x)
- <strong>Heckeberg NS</strong>. The Systematics of Cervidae. *Deer of the World*. [link](https://doi.org/10.1007/978-3-031-17756-9_1)
- Catalán A, Gygax D, Candolin U, Tusso S, Duchen P and <strong>Höhna S</strong>. Sex-biased migration and demographic history of the big European firefly *Lampyris noctiluca*. *Molecular Biology and Evolution*,  42:6, msaf123,. [link](https://doi.org/10.1093/molbev/msaf123)
- <strong>Höhna S</strong> and Catalán A. Bayesian StairwayPlot for Inferring Single Population Demographic Histories from Site Frequency Spectra. *Molecular Ecology Resources*, 25:e14087. [link](https://doi.org/10.1111/1755-0998.14087)
- <strong>Darlim G</strong> and <strong>Höhna S</strong>. The effects of cryptic diversity on diversification dynamics analyses in Crocodylia. *Proceedings of the Royal Society B*, 292 (2043): 20250091. [link](https://doi.org/10.1098/rspb.2025.0091)
- Olroyd SL and <strong>Kopperud BT</strong>. Allometry of sound reception structures and evidence for a mandibular middle ear in non-mammalian synapsids. *Evolution*, 79(6), 905-921. [link](https://doi.org/10.1093/evolut/qpaf041)
- <strong>Höhna S</strong>, Lower SE, Duchen P and Catalán A. Robustness of divergence time estimation despite gene tree estimation error: a case study of fireflies (Coleoptera: Lampyridae). *Systematic Biology*, 74(2), 335-348. [link](https://doi.org/10.1093/sysbio/syae065)
- Mulvey LPA, May MR, Brown JM, <strong>Höhna S</strong>, Wright AM and Warnock RCM. Assessing the Adequacy of Morphological Models Using Posterior Predictive Simulations. *Systematic Biology*, 74(1), 34-52. [link](https://doi.org/10.1093/sysbio/syae055)
- <strong>Capobianco A</strong>, Zouhri S and Friedman M. A long-snouted marine bonytongue (Teleostei: Osteoglossidae) from the early Eocene of Morocco and the phylogenetic affinities of marine osteoglossids. *Zoological Journal of the Linnean Society*, 203(1), zlae015. [link](https://doi.org/10.1093/zoolinnean/zlae015)


<h3>2024</h3>
- <strong>Capobianco A</strong>, Friedman M. Fossils indicate marine dispersal in osteoglossid fishes, a classic example of continental vicariance. *Proceedings of the Royal Society B*, 291:20241293. [link](https://doi.org/10.1098/rspb.2024.1293)
- <strong>Khakurel B</strong>, Grigsby C, Tran TD, Zariwala J, <strong>Höhna S</strong>, Wright AM. The fundamental role of character coding in Bayesian morphological phylogenetics. *Systematic Biology*, 73(5), 861-871. [link](https://doi.org/10.1093/sysbio/syae033)
- Bartoszek K, <strong>Clarke JT</strong>, Fuentes-González J, Mitov V, Pienaar J, Piwczyński M, Puchałka R, Spalik K, Voje KL. Fast mvSLOUCH: Multivariate Ornstein–Uhlenbeck-based models of trait evolution on large phylogenies. *Methods in Ecology and Evolution*, 15(9), 1507-1515. [link](https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.14376)
- <strong>Clarke JT</strong> and Davis, RB. Salinity plays a limited role in determining rates of size evolution in fishes globally across multiple scales. *Global Ecology and Biogeography*, 33(9), e13883. [link](https://onlinelibrary.wiley.com/doi/full/10.1111/geb.13883)
- Karapunar B, <strong>Höhna S</strong> and Nützel A. Phylogeny of the longest existing gastropod clade (Pleurotomariida) reconstructed with Bayesian and parsimony methods and its implications on gastropod shell characters, *Journal of Systematic Palaeontology*, 22(1), 2384141. [link](https://doi.org/10.1080/14772019.2024.2384141)
- <strong>Billenstein RJ</strong>, <strong>Höhna S</strong>. Comparison of Bayesian Coalescent Skyline Plot Models for Inferring Demographic Histories. *Molecular Biology and Evolution*, 41(5), msae073. [link](https://academic.oup.com/mbe/article/41/5/msae073/7648822)
- <strong>Höhna S</strong>, <strong>Hsiang AY</strong>. Sequential Bayesian phylogenetic inference. *Systematic Biology*, 73(4), 704--721. [link](https://doi.org/10.1093/sysbio/syae020)
- <strong>Smith K</strong>, Ayres D, Neumaier R, Wörheide G, <strong>Höhna S</strong>. Bayesian Phylogenetic Analysis on multi-core Compute Architectures: Implementation and evaluation of BEAGLE in RevBayes with MPI. *Systematic Biology*, 73(2), 455-469. [link](https://academic.oup.com/sysbio/article/73/2/455/7591007)
- Foerster SIA, <strong>Clarke JT</strong>, Õunap E, Teder T, Tammaru T. A Comparative Study of Body Size Evolution in Moths: Evidence of Correlated Evolution with Feeding and Phenology-Related Traits. *Journal of Evolutionary Biology*, 37(8), 891–904. [link](https://academic.oup.com/jeb/article-abstract/37/8/891/7689394)
- Sepp T, Baines C, Kreitsberg R, Scharsack JP, Nogueira P, Lang T, Fort J, Sild E, <strong>Clarke JT</strong>, Tuvikene A, Meitern R. Differences on the level of hepatic transcriptome between two flatfish species in response to liver cancer and environmental pollution levels. *Comparative Biochemistry and Physiology Part C: Toxicology & Pharmacology*, 275, 109781. [link](https://www.sciencedirect.com/science/article/pii/S1532045623002363)
- <strong>Fabreti LG</strong>, Coghill LM, Thomson RC, <strong>Höhna S</strong>, Brown JM. The Expected Behaviors of Posterior Predictive Tests and Their Unexpected Interpretation. Molecular Biology and Evolution, 41(3), msae051. [link](https://academic.oup.com/mbe/article/41/3/msae051/7618542)
- Tsuboi M, <strong>Kopperud BT</strong>, Matschiner M, Grabowski M, Syrowatka C, Pélabon C, Hansen TF. Antler Allometry, the Irish Elk and Gould Revisited. *Evolutionary Biology*, 51, 149-165. [link](https://link.springer.com/article/10.1007/s11692-023-09624-1)
- Martínez-Gómez J, Song MJ, Tribble CM, <strong>Kopperud BT</strong>, Freyman WA, <strong>Höhna S</strong>, Specht CD, Rothfels CJ. Commonly used Bayesian diversification methods lead to biologically meaningful differences in branch-specific rates on empirical phylogenies. *Evolution Letters*, 8(2), 189-199. [link](https://academic.oup.com/evlett/article/8/2/189/7334365)


<h3>2023</h3>
- <strong>Darlim G</strong>, Suraprasit K, Chaimanee Y, Tian P, Yamee C, Rugbumrung M., ... and Rabi M. An extinct deep-snouted Alligator species from the Quaternary of Thailand and comments on the evolution of crushing dentition in alligatorids. *Scientific Reports*, 13(1), 10406. [link](https://www.nature.com/articles/s41598-023-36559-6)
- <strong>Heckeberg NS</strong>, Zachos FE, Kierdorf U. Antler tine homologies and cervid systematics: A review of past and present controversies with special emphasis on Elaphurus davidianus. *The Anatomical Record*, 306(1), 5-28. [link](https://anatomypubs.onlinelibrary.wiley.com/doi/full/10.1002/ar.24956)
- Meng Y, Davison J, <strong>Clarke JT</strong>, Zobel M, Gerz M, Moora M, Öpik M, Bueno CG. Environmental modulation of plant mycorrhizal traits in the global flora. *Ecology Letters*, 26(11), 1862-1876. [link](https://onlinelibrary.wiley.com/doi/full/10.1111/ele.14309)
- <strong>Fabreti L</strong> and <strong>H&ouml;hna S</strong>. Nucleotide Substitution Model Selection Is Not Necessary for Bayesian Inference of Phylogeny With Well-Behaved Priors, *Systematic Biology*, 72(6), 1418-1432. [link](https://doi.org/10.1093/sysbio/syad041)
- Grabowski M, Pienaar J, Voje KJ, Andersson S, Fuentes-González J, <strong>Kopperud BT</strong>, Moen DS, Tsuboi M, Uyeda J and Hansen TF. A Cautionary Note on “A Cautionary Note on the Use of Ornstein Uhlenbeck Models in Macroevolutionary Studies”. *Systematic Biology*, 72(4), 955-963, [link](https://doi.org/10.1093/sysbio/syad012)
- Grabowski M, <strong>Kopperud BT</strong>, Tsuboi M and Hansen TF. Both diet and sociality affect primate brain-size evolution. *Systematic Biology*, 72(2), 404-418, [link](https://doi.org/10.1093/sysbio/syac075)
- Juravel K, Porras L, <strong>H&ouml;hna S</strong>, Pisani D and Wörheide G. Exploring genome gene content and morphological analysis to test recalcitrant nodes in the animal phylogeny, *PLoS one*, 18 (3): e0282444. [link](https://doi.org/10.1371/journal.pone.0282444)
- <strong>Kopperud BT</strong>, Magee AF, <strong>H&ouml;hna S</strong>. Rapidly changing speciation and extinction rates can be inferred in spite of nonidentifiability. *Proceedings of the National Academy of Sciences*, 120(7), e2208851120. [link](https://www.pnas.org/doi/abs/10.1073/pnas.2208851120)


<h3>2022</h3>
- <strong>H&ouml;hna S</strong>, <strong>Kopperud BT</strong>, Magee AF. CRABS: Congruent Rate Analyses in Birth-death Scenarios, *Methods in Ecology and Evolution*, 13, 2709–2718, [link](https://doi.org/10.1111/2041-210X.13997)
- Catalán A, <strong>H&ouml;hna S</strong>, Lower SE, Duchen P. Inferring the demographic history of the North American firefly *Photinus pyralis*, *Journal of Evolutionary Biology*, 35, 1488-1499, [link](https://doi.org/10.1111/jeb.14094)
- Borges R, Boussau B, <strong>H&ouml;hna S</strong>, Pereira RJ, Kosiol C. Polymorphism-aware estimation of species trees and evolutionary forces from genomic sequences with RevBayes, *Methods in Ecology and Evolution*, 13, 2339-2346, [link](https://doi.org/10.1111/2041-210X.13980)
- <strong>Kopperud BT</strong>, Lidgard S and Liow LS. Enhancing georeferenced biodiversity inventories: automated information extraction from literature records reveal the gaps, *PeerJ*, 10:e13921, [link](https://doi.org/10.7717/peerj.13921)
- Barido-Sottani J, Justison JA, Borges R, Brown JM, Dismukes W, Petrucci BR, <strong>Fabreti L</strong>, <strong>H&ouml;hna S</strong>, Landis MJ, Lewis, PO, May MR, Mendes FK, Pett W, Redelings BD, Tribble CM, Wright AM, Zenil-Ferguson R and Heath TA. Lessons learned from organizing and teaching virtual phylogenetics workshops. *The Bulletin of the Society of Systematic Biologists*, 1(2):8425, [link](https://ssbbulletin.org/index.php/bssb/article/download/8425/7557).
- Sz&ouml;ll&#337;si GJ, <strong>H&ouml;hna S</strong>, Williams TA, Schrempf D, Daubin V and Boussau B. Relative time constraints improve molecular dating, *Systematic Biology*, 71(4), 797--809, [link](https://doi.org/10.1093/sysbio/syab084)
- Tribble CM, Freyman WA, Landis MJ, Lim JY, Barido-Sottani J, <strong>Kopperud BT</strong>, <strong>H&ouml;hna S</strong> and May MR. RevGadgets: an R Package for visualizing Bayesian phylogenetic analyses from RevBayes, *Methods in Ecology and Evolution*, 13:314-323, [link](https://doi.org/10.1111/2041-210X.13750)
- Palazzesi L, Hidalgo O, Barreda VD, Forest F, and <strong>H&ouml;hna S</strong>. The rise of grasslands is linked to atmospheric CO<sub>2</sub> decline in the late paleogene. *Nature Communications*, 13, 293, [link](https://doi.org/10.1038/s41467-021-27897-y)
- <strong>Fabreti L</strong>, <strong>H&ouml;hna S</strong>. Convergence assessment for bayesian phylogenetic analysis using mcmc simulation, *Methods in Ecology and Evolution*, 13:77–90, [link](https://doi.org/10.1111/2041-210X.13727)


<h3>2021</h3>
- <strong>H&ouml;hna S</strong>, Landis MJ and Huelsenbeck JP. Parallel power posterior analyses for fast computation of marginal likelihoods in phylogenetics, *PeerJ*, 9:e12438, [link](https://doi.org/10.7717/peerj.12438)
- Orsi WD, Magritsch T, Vargas S, Coskun &Ouml;K, Vuillemin A, <strong>H&ouml;hna S</strong>, W&ouml;rheide G, D'Hondt S, Shapiro BJ, Carini P. Genome evolution in bacteria isolated from million-year-old subseafloor sediment, *mBio*, 12:e01150--21, [link](https://doi.org/10.1128/mBio.01150-21).


<h3>2020</h3>
- Magee AF, <strong>H&ouml;hna S</strong>, Vasylyeva TI, Leaché AD, and Minin VN. Locally adaptive Bayesian birth-death model successfully detects slow and rapid rate shifts, *PLoS Comp. Biol.*, 16(10): e1007999, [link](https://doi.org/10.1371/journal.pcbi.1007999)


<h3>2019</h3>
- Catalán A, Briscoe AD and <strong>H&ouml;hna S</strong>. Drift and Directional Selection Are the Evolutionary Forces Driving Gene Expression Divergence in Eye and Brain Tissue of Heliconius Butterflies, *Genetics*, 213(2), 581-594, [link](https://doi.org/10.1534/genetics.119.302493)
- Freyman WA and <strong>H&ouml;hna S</strong>. Stochastic Character Mapping of State-Dependent Diversification Reveals the Tempo of Evolutionary Decline in Self-Compatible Onagraceae Lineages, *Systematic Biology*, 68(3), 505-519, [link](https://academic.oup.com/sysbio/article-abstract/68/3/505/5200730)
- Silvestro D, Tejedor MF, Serrano-Serrano ML, Loiseau O, Rossier V, Rolland J, Zizka A, <strong>H&ouml;hna S</strong>, Antonelli A and Salamin N. Early Arrival and Climatically-Linked Geographic Expansion of New World Monkeys from Tiny African Ancestors, *Systematic Biology*, 68 (1), 78-92, [link](https://academic.oup.com/sysbio/article/68/1/78/5040681)
- <strong>Hsiang AY</strong>, Brombacher A, Rillo MC, Mleneck-Vautravers MJ, Conn S, Lordsmith S, Jentzen A, Henehan MJ, Metcalfe B, Fenton I, Wade BS, Fox L, Meilland J, Davis CV, Baranowski U, Groeneveld J, Edgar KM, Movellan A, Aze T, Dowsett H, Miller G, Rios N, Hull PM. Endless Forams: >34,000 modern planktonic foraminiferal images for taxonomic training and automated species recognition using convolutional neural networks, *Paleoceanography and Paleoclimatology*, 34, [link](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2019PA003612)
- Field DJ, Berv JS, <strong>Hsiang AY</strong>, Lanfear R, Landis MJ, Dornburg A. Timing the extant avian radiation: The rise of modern birds, and the importance of modelling molecular rate variation, *PeerJ Preprints*, 7, e27521v1, [link](https://peerj.com/preprints/27521/)


<h3>2018</h3>
- Condamine FL, Rolland J, <strong>H&ouml;hna S</strong>, Sperling FAH and Sanmartin I. Testing the Role of the Red Queen and Court Jester as Drivers of the Macroevolution of Apollo Butterflies, *Systematic Biology*, 67(6), 940-964, [link](https://academic.oup.com/sysbio/article-abstract/67/6/940/4847851)
- Martin CH and <strong>H&ouml;hna S</strong>. New evidence for the recent divergence of Devil's Hole pupfish and the plausibility of elevated mutation rates in endangered taxa, *Molecular Ecology*, 27(4), 831-838, [link](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.14404)
- <strong>H&ouml;hna S</strong>, Coghill LM, Mount G, Thomson R and Brown JM. P<sup>3</sup>: Phylogenetic Posterior Prediction in RevBayes, *Molecular Biology and Evolution*, 35(4), 1028-1034, [link](https://academic.oup.com/mbe/article/35/4/1028/4616601)
- Freyman WA and <strong>H&ouml;hna S</strong>. Cladogenetic and Anagenetic Models of Chromosome Number Evolution: a Bayesian Model Averaging Approach, *Systematic Biology*, 67(2), 195-215, [link](https://academic.oup.com/sysbio/article/67/2/195/4037179)


<h3>2017</h3>
- <strong>H&ouml;hna S</strong>, Landis MJ and Heath TA. Phylogenetic Inference using RevBayes. *Current Protocols in Bioinformatics*, 57:6.16.1-6.16.34, [link](http://onlinelibrary.wiley.com/doi/10.1002/cpbi.22/full)
- Martin CH, <strong>S H&ouml;hna</strong>, Crawford JE, Turner BJ, Richards EJ and Simons LH. The complex effects of demographic history on the estimation of substitution rate: concatenated analysis results in no more than 2-fold overestimation, *Proceedings of the Royal Society B*, 284: 20170537, [link](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2017.0537)


<h3>2016</h3>
- Moore BR, <strong>H&ouml;hna S</strong>, May MR, Rannala B and Huelsenbeck JP. Critically evaluating the theory and performance of Bayesian analysis of macroevolutionary mixtures. *Proceedings of the National Academy of Sciences*, 113(34), 9569-9574, [link](https://www.pnas.org/content/113/34/9569.short)
- <strong>H&ouml;hna S</strong>, Landis MJ, Heath TA, Boussau B, Moore BR, Lartillot N, Huelsenbeck JP and Ronquist F. RevBayes: Bayesian Phylogenetic Inference Using Graphical Models and an Interactive Model-Speci cation Language, *Systematic Biology*, 65(4), 726-736, [link](https://academic.oup.com/sysbio/article/65/4/726/1753608/RevBayes-Bayesian-Phylogenetic-Inference-Using)
- Conroy CR, Patton JL, Lim M, Phuong M, Parmenter B and <strong>H&ouml;hna S</strong>. Following the rivers: historical reconstruction of California voles (Microtus californicus, Muridae, Rodentia) in the deserts of eastern California, *Biological Journal of the Linnean Society*, 119(1), 80–98, [link](http://onlinelibrary.wiley.com/doi/10.1111/bij.12808/full)
- <strong>H&ouml;hna S</strong>, May MR and Moore BR. TESS: an R package for efficiently simulating phylogenetic trees and performing Bayesian inference of lineage diversification rates, *Bioinformatics*, 32(5): 789-791, [link](https://academic.oup.com/bioinformatics/article/32/5/789/1744433/TESS-an-R-package-for-efficiently-simulating)
