---
layout: page
title: Research
permalink: /research/
---


<header class="page-header">
  <h2 class="page-title">Research</h2>
  <br>
</header>

<div class="center">

<p class="intro-text">
  Our research is focused on several aspects of theoretical and computational phylogenetics. Here we list the main directions, but see the <a href="/publications" target="_blank"><strong>publications page</strong></a> for more details.
</p>

<div class="projects-container">

  <!-- Section 1 -->
  <div class="project">
    <img class="project-image floatleft" src="/assets/images/Figure_1_v2.png" alt="Robust Estimation of Gene Trees">
    <div class="project-content">
        <h3>Robust Estimation of Gene Trees</h3>
        <p class="project-description">
            Gene trees provide fundamental information about the processes in genetic evolution which is contained in the variation of their topology and branching times.
            In this work we are focused on how to reliably estimate gene trees by developing more robust statistical methods.
        </p>
    </div>
  </div>

  <br><br>

  <!-- Section 2 -->
  <div class="project">
    <img class="project-image floatright" src="/assets/images/Figure_2_v2.png" alt="Understanding Discordance Between Species Trees and Gene Trees">
    <div class="project-content">
        <h3>Understanding Discordance Between Species Trees and Gene Trees</h3>
        <p class="project-description">
            The following biological scenarios can cause incongruent gene trees: (a) simple population genetic processes (i.e., the coalescent), (b) migration and thus gene-flow between species/populations , and (c) gene-duplication and gene-loss.
            We are working on more realistic models that incorporate different causes of gene-tree incongruence such as the multispecies coalescent with migration model.
            This work also entails new algorithm development and theory design for more efficient computation of species trees from thousands of gene trees.
        </p>
    </div>
  </div>

  <br><br>

  <!-- Section 3 -->
  <div class="project">
    <img class="project-image floatleft" src="/assets/images/Figure_4.png" alt="Inference of Macroevolutionary Processes">
    <div class="project-content">
        <h3>Inference of Macroevolutionary Processes</h3>
        <p class="project-description">
            Our ultimate goal is to infer diversification rates through time and among lineages and identify correlations to genetic, phenotypic and/or environmental factors that impact diversification rate.
            For example, we developed a statistical method to identify correlations between rates of diversification and environmental variables, such as changes in atmospheric CO2.
            In our future work, we want to incorporate fossil occurrence information in our models of lineage diversification.
        </p>
    </div>
  </div>

  <br><br>

  <!-- Section 4 -->
  <div class="project">
    <img class="project-image floatright" src="/assets/images/trait_evolution.png" alt="The State-Dependent OU Model">
    <div class="project-content">
        <h3>The State-Dependent OU Model</h3>
        <p class="project-description">
            Joint inference of continuous trait adaptation and selective regime history using the state-dependent Ornstein-Uhlenbeck model.
            Trait evolution studies the evolution of homologous characters across phylogeny. These characters are at above-sequence level, including but not limited to gene expression, physiological, and morphological data.
            Many phylogenetic comparative methods (PCMs) have been developed to study trait evolution, yet to what extent these models reflect realistic evolutionary processes is still under debate.
            Here, we have developed the state-dependent Ornstein-Uhlenbeck model, a Bayesian approach to jointly infer the evolution of a continuous trait and the discrete selective regimes.
        </p>
    </div>
  </div>

  <br><br>

  <!-- Section 5 -->
  <div class="project">
    <img class="project-image floatleft" src="/assets/images/gene_family.jpg" alt="Gene Family Evolution in Fireflies">
    <div class="project-content">
        <h3>Modeling gene expression evolution in fireflies</h3>
        <p class="project-description">
          Gene expression is a key driver of trait variation, particularly among closely related species. This project aims to develop innovative methods to model gene expression evolution using Brownian motion and Ornstein-Uhlenbeck processes. Specially, we focus on within-species variance, a critical yet underexplored aspect of gene expression.
          The firefly family (Lampyridae) serves as a novel study system due to their recurrent sexual dimorphism across the phylogeny. As sexual dimorphism is inherently linked to sex-biased gene expression, this makes fireflies an ideal model for investigating sex-biased gene expression evolution.
        </p>
    </div>
  </div>

  <br><br>

  <!-- Section 6 -->
  <div class="project">
    <img class="project-image floatright" src="/assets/images/MacDrive.png" alt="MacDrive">
    <div class="project-content">
        <h3>MacDrive</h3>
        <p class="project-description">
            Biodiversity is modeled by the process of speciation and extinction. There is clear evidence from both living and extinct species that biodiversity is extremely variable through time and among species.
            However, we still do not know what factors drive speciation and extinction rates on a macroevolutionary level (that is, beyond species boundaries).
            The goal of this project is to combine statistical, computational, neontological – i.e. relating to species living today – and paleobiological approaches to study macroevolutionary dynamics.
        </p>
    </div>
  </div>

</div>

</div>
